storage/interface.go

```diff
@@ -56,6 +56,13 @@ type MetricStore interface {
 	// undefinded state). If nil is returned, the MetricStore cannot be
 	// "restarted" again, but it can still be used for read operations.
 	Shutdown() error
+	// Returns true, nil if the MetricStore is currently working as expected
```

Comment:
Please start the doc comment with `Healthy`, as above, i.e. `// Healthy returns true...`.

Reply:
done